src/config/index.ts

```diff
@@ -39,6 +39,9 @@ export const config = {
   databaseUrl: String(process.env.DATABASE_URL),
   databaseSSL: Boolean(Number(process.env.DATABASE_SSL)),
   readReplicaDatabaseUrl: String(process.env.READ_REPLICA_DATABASE_URL || process.env.DATABASE_URL),
+  korsoReplicaDatabaseUrl: String(
```

Comment:
What's korso?

Reply:
lol. It was a new name if priori, which couldn't pass legal checks.